src/shared/services/bolt/boltMappings.js

```diff
@@ -219,3 +219,31 @@ export const retrieveFormattedUpdateStatistics = (result) => {
 export const flattenProperties = (rows) => {
   return rows.map((row) => row.map((entry) => (entry && entry.properties) ? entry.properties : entry))
 }
+
+export const applyGraphTypes = (item) => {
```

Comment:
I'd like to see this function tested, so we don't introduce regressions if we need to make changes in it in the future.